network/manager.go

```diff
@@ -354,11 +403,56 @@ func (nm *networkManager) CreateEndpoint(cli apipaClient, networkID string, epIn
 	return nil
 }
 
+// UpdateEndpointState will make a call to CNS updatEndpointState API in the stateless CNI mode
+// It will add HNSEndpointID or HostVeth name to the endpoint state
+func (nm *networkManager) UpdateEndpointState(ep *endpoint) error {
+	cnilogger.Logger.Info("Calling cns updateEndpoint API with ", zap.String("containerID: ", ep.ContainerID), zap.String("HnsId: ", ep.HnsId), zap.String("HostIfName: ", ep.HostIfName))
+	response, err := nm.CnsClient.UpdateEndpoint(context.TODO(), ep.ContainerID, ep.HnsId, ep.HostIfName)
+	if err != nil {
+		cnilogger.Logger.Error("Update endpoint API returend with error", zap.Error(err))
+		return errors.Wrapf(err, "Get endpoint API returend with error")
```

Comment:
The error description is saying `Get endpoint` instead of `Update endpoint`

Reply:
addressed in the next commit